client/packages/invoices/src/Prescriptions/DetailView/hooks/usePrinter.tsx

```diff
@@ -61,8 +61,7 @@ export const usePrintLabels = () => {
       })
       .catch(e => {
         error(`${t('error.printing-label')}: ${e.message}`)();
-      })
-      .finally(() => setIsPrintingLabels(false));
+      });
     setIsPrintingLabels(false);
```

Comment:
Heh my bad if not clear - I think we should keep the finally!

The problem is that setIsPrintingLabels(false) will run immediately here, and the promise above will run async in the background. So the state essentially is never `true`. `Finally()` will run after the last `then` or `catch`, so will correctly call `setIsPrintingLabels(false)` after the request/promise has completed